### PR TITLE
PIN-5242: Reprocess files

### PR DESCRIPTION
**Pull request by diego-lon**
This PR implements the functionality of reprocessing files retrieved from an S3 bucket. It includes:

- A GitHub Action to manually trigger the process by specifying the S3 path.
- Node.js setup with dependencies managed via pnpm.
- Environment variables configured for AWS and queue interactions.
- Core functionality that reprocesses the file and integrates with AWS S3 and SQS.
- Tests

### CODE REVIEW

File: package.json
Comment (eduwardo) [inline package.json:1]: We should enforce strict versioning and avoid unintended updates that could introduce compatibility issues, also update dependencies where it is possibile, ex. "@types/node": "^16.0.0" to 20 version

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+!.vscode/*.code-snippets
+
+# Local History for Visual Studio Code
+.history/
+.turbo/
+
+**/dist
+packages/models/src/gen
+packages/*/src/model/generated
+packages/clients/*/src/model/generated
```
Comment (eduwardo) [inline .gitignore]: Not necessary

**Review by eduwardo — COMMENTED** [review]
Overall ok, minor fix

**Review by lorenzo-pa — APPROVED** [review]

**Comment by diego-lon** [comment]
Closed. It will be split into two PRs: one for GitHub workflows and the other for the application.

**Pull request closed without merging**